tests/Ardalis.HttpClientTestExtensions.Tests/HttpClientPutExtensionMethodsTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Ardalis.HttpClientTestExtensions.Api.Dtos;
+using Shouldly;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Ardalis.HttpClientTestExtensions.Tests;
+
+public class HttpClientPutExtensionMethodsTests : IClassFixture<CustomWebApplicationFactory>
+{
+  private readonly HttpClient _client;
+  private readonly ITestOutputHelper _outputHelper;
+
+  public HttpClientPutExtensionMethodsTests(CustomWebApplicationFactory factory, ITestOutputHelper outputHelper)
+  {
+    _client = factory.CreateClient();
+    _outputHelper = outputHelper;
+  }
+
+  [Fact]
+  public async Task PostAndEnsureNotFoundTestAsync()
+  {
+    var dto = new CountryDto();
+    var content = new StringContent(JsonSerializer.Serialize(dto), Encoding.UTF8, "application/json");
+    var response = await _client.PutAndEnsureNotFoundAsync("/wrongendpoint", content, _outputHelper);
+
+    response.StatusCode.ShouldBe(HttpStatusCode.NotFound);
```

Comment:
Not needed;
Maybe keep one of these with a comment saying it's not needed but shown to demonstrate it's available to use?

Reply:
ok

Reply:
Done